site/docs/tutorial/final_plan.json

```
@@ -0,0 +1,381 @@
+{
+  "version": { "minor": 18, "producer": "validator-test" },
```

The version field was renamed in 0.19.x via https://github.com/substrait-io/substrait/pull/362:
```suggestion
  "version": { "minorNumber": 19, "producer": "validator-test" },
```